openzeppelin/token/erc20/library.cairo

```diff
@@ -2,11 +2,13 @@
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
 from starkware.starknet.common.syscalls import get_caller_address
-from starkware.cairo.common.math import assert_not_zero
+from starkware.cairo.common.math import assert_not_zero, assert_lt
 from starkware.cairo.common.uint256 import (
     Uint256, uint256_add, uint256_sub, uint256_le, uint256_lt, uint256_check
 )
 
+const UINT8_MAX = 256
+
```

Comment:
Great addition! I'm thinking maybe this should be stored in the `utils/constants` directory. I doubt we'll be using it elsewhere in the near term, but it might make the ERC20 lib look cleaner with the import. What do you think?

Reply:
Sounds good to me 👍